realm/realm-library/build.gradle

```diff
@@ -30,6 +30,8 @@ if (!ext.coreArchiveDir) {
 ext.coreArchiveFile = rootProject.file("${ext.coreArchiveDir}/realm-sync-android-${project.coreVersion}.tar.gz")
 ext.coreDistributionDir = file("${projectDir}/distribution/realm-core/")
 ext.coreDir = file("${project.coreDistributionDir.getAbsolutePath()}/core-${project.coreVersion}")
+ext.ccachePath = project.findProperty('ccachePath') ?: System.getenv('REALM_CCACHE_PATH')
```

Comment:
Actually... why using custom environment variables? The CMake toolchain already accepts `NDK_CCACHE`. Could we just use that and introduce `NDK_LCACHE` maybe?